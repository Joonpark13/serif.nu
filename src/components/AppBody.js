import React from 'react';
import { makeStyles } from '@material-ui/styles';
import Grid from '@material-ui/core/Grid';
import CalendarView from './calendar/CalendarView';
import SidebarView from './sidebar/SidebarView';

const useStyles = makeStyles({
  calendar: {
    overflow: 'auto',
  },
});

export default function AppBody() {
  const classes = useStyles();
  return (
    <Grid container>
      <Grid item md={9} sm={12} xs={12} className={classes.calendar}>
        <CalendarView />
      </Grid>

      <Grid item md={3} sm={12} xs={12}>
        <SidebarView />
      </Grid>
    </Grid>
  );
}
