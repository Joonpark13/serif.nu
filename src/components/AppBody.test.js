import React from 'react';
import { shallow } from 'enzyme';
import AppBody from './AppBody';

jest.mock('images/hero-image.jpg');

describe('AppBody', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<AppBody />);

    expect(wrapper.get(0)).toMatchSnapshot();
  });
});
