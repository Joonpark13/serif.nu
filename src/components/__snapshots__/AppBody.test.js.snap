// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBody should render correctly 1`] = `
<ForwardRef(WithStyles)
  container={true}
>
  <ForwardRef(WithStyles)
    className="makeStyles-calendar-1"
    item={true}
    md={9}
    sm={12}
    xs={12}
  >
    <CalendarView />
  </ForwardRef(WithStyles)>
  <ForwardRef(WithStyles)
    item={true}
    md={3}
    sm={12}
    xs={12}
  >
    <SidebarView />
  </ForwardRef(WithStyles)>
</ForwardRef(WithStyles)>
`;
